frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestReviewStatus/ChangeRequestReviewStatus.tsx

```diff
@@ -194,6 +203,69 @@ const Applied = () => {
     );
 };
 
+interface IScheduledProps {
+    scheduledDate?: string;
+}
+const Scheduled = ({ scheduledDate }: IScheduledProps) => {
+    const theme = useTheme();
+
+    if (!scheduledDate) {
+        return null;
+    }
+
+    const getBrowserTimezone = (): string => {
+        const offset = -new Date().getTimezoneOffset();
+        const hours = Math.floor(Math.abs(offset) / 60);
+        const minutes = Math.abs(offset) % 60;
+        let sign = '+';
+        if (offset < 0) {
+            sign = '-';
+        }
+
+        // Ensure that hours and minutes are two digits
+        const zeroPaddedHours = hours.toString().padStart(2, '0');
+        const zeroPaddedMinutes = minutes.toString().padStart(2, '0');
+
+        return `UTC${sign}${zeroPaddedHours}:${zeroPaddedMinutes}`;
+    };
```

Comment:
What is this doing? Feels like this should be extracted into a separate function (or even file) and tested. Or if we're just getting the timezone, could we use `Intl.DateTimeFormat().resolvedOptions().timeZone` instead? It won't give the offset, but it will give the time zone, which might be more important. There might also be cases where it's more correct, such as if you're thinking about daylight saving time. The `getBrowserTimezone` won't account for that, but using `Intl.DateTimeFormat().resolvedOptions().timeZone` will, because you're always working with the user's current time zone, and not its offset from UTC (which can change).

Reply:
But thinking about it: how do we store times? UTC-times? Regardless, think we should try to avoid doing as much time zone math as possible

Reply:
`Intl.DateTimeFormat().resolvedOptions().timeZone` would give eg. Asia/Pasific or GMT+2

I couldn't find a way to format it as requested

`getTimezoneOffset` returns the offset in minutes from GMT


`hours` calculated from minutes / 60

`minutes` any remainder from the hours calculation (eg. India is UTC+05:30)

Finally the desired format is created by ensuring 2 digits (padding with 0) and putting the different parts together

Reply:
I will move to its own file and write a few tests thanks

Reply:
Yeah, so for me, `Intl.DateTimeFormat().resolvedOptions().timeZone` gives "Europe/Oslo". 

You're right in that it doesn't give you a UTC offset. But I think that's better. The scary thing about using offsets like this is that the offset you're calculating here is based on the current time, while the time could be in the future. So if that goes over daylight saving times, I think it would be wrong.

Say you want it to happen at 10:00 UTC-2 in two weeks. But there's a clock change in between, so you're currently at UTC-3. So you think you need to set it to be at 11:00 in two weeks. That way it says 11:00 UTC-3, but it's actually 11:00 UTC-2 because the date picker knows about this. In this case 10:00 Europe/Oslo is more accurate.

Reply:
```The time-zone offset is the difference, in minutes, between UTC and local time. Note that this means that the offset is positive if the local timezone is behind UTC and negative if it is ahead. For example, for time zones that are ahead of UTC, the value would be negative because local time is added to the UTC to convert it to local time.```

Reply:
Sure, I agree with that, but I don't think it adequately handles time changes, such as DST. And I'm afraid that's gonna cause issues down the line.